src/Network/Network.Test/ScenarioTests/LoadBalancerTests.ps1

```diff
@@ -1096,6 +1096,172 @@ function Test-LoadBalancerChildResource
     }
 }
 
+<#
```

Comment:
May need to update the test code for `Test-LoadBalancerChildResource`.

Reply:
Already test inboundNatRule v2 in Test-LoadBalancerInboundNatRuleV2-InternalLB and Test-LoadBalancerInboundNatRuleV2. If update Test-LoadBalancerChildResource, that will just add the same code